Source/Microsoft.Teams.Apps.FAQPlusPlus.Configuration/Microsoft.Teams.Apps.FAQPlusPlus.Configuration.csproj

```diff
@@ -274,6 +274,7 @@
     <Compile Include="Global.asax.cs">
       <DependentUpon>Global.asax</DependentUpon>
     </Compile>
+    <Compile Include="GlobalSuppressions.cs" />
```

Comment:
Not sure what we intended to do with this change-- we seem to have removed the file in line 263, and then added it back here?

Reply:
Removed the file and moved the suppression using #pragma.